braph2genesis/measures/_MultilayerModlrty.gen.m

```diff
@@ -0,0 +1,138 @@
+%% ¡header!
+MultilayerModlrty < MultilayerCommunity  (m, multilayer modularity) is the graph multilayer modularity.
+
+%%% ¡description!
+The multilayer modularity of a multilayer graph is the multilayer 
+quality function Q of the resulting partition of the multilayer network.  
+
+%% ¡layout!
+
+%%% ¡prop!
+%%%% ¡id!
+MultilayerModlrty.ID
+%%%% ¡title!
+Measure ID
+
+%%% ¡prop!
+%%%% ¡id!
+MultilayerModlrty.LABEL
+%%%% ¡title!
+Measure NAME
+
+%%% ¡prop!
+%%%% ¡id!
+MultilayerModlrty.G
+%%%% ¡title!
+Graph
+
+%%% ¡prop!
+%%%% ¡id!
+MultilayerModlrty.M
+%%%% ¡title!
+MultilayerModlrty
+
+%%% ¡prop!
+%%%% ¡id!
+MultilayerModlrty.PFM
+%%%% ¡title!
+Measure Plot
+
+%%% ¡prop!
+%%%% ¡id!
+MultilayerModlrty.NOTES
+%%%% ¡title!
+Measure NOTES
+
+%%% ¡prop!
+%%%% ¡id!
+MultilayerModlrty.COMPATIBLE_GRAPHS
+%%%% ¡title!
+Compatible Graphs
+
+%% ¡props_update!
+
+%%% ¡prop!
+NAME (constant, string) is the name of the multilayer modularity.
+%%%% ¡default!
+'MultilayerModlrty'
+
+%%% ¡prop!
+DESCRIPTION (constant, string) is the description of the multilayer modularity.
+%%%% ¡default!
+'The multilayer modularity is the homogeneity of the number of inward neighbours of a node across the layers.'
+
+%%% ¡prop!
+TEMPLATE (parameter, item) is the template of the multilayer modularity.
+%%%% ¡settings!
+'MultilayerModlrty'
+
+%%% ¡prop!
+ID (data, string) is a few-letter code of the multilayer modularity.
+%%%% ¡default!
+'MultilayerModlrty ID'
+
+%%% ¡prop!
+LABEL (metadata, string) is an extended label of the multilayer modularity.
+%%%% ¡default!
+'MultilayerModlrty label'
+
+%%% ¡prop!
+NOTES (metadata, string) are some specific notes about the multilayer modularity.
+%%%% ¡default!
+'MultilayerModlrty notes'
+
+%%% ¡prop!
+SHAPE (constant, scalar) is the measure shape __Measure.NODAL__.
+%%%% ¡default!
+Measure.GLOBAL
+
+%%% ¡prop!
+SCOPE (constant, scalar) is the measure scope __Measure.UNILAYER__.
+%%%% ¡default!
+Measure.SUPERGLOBAL
+
+%%% ¡prop!
+PARAMETRICITY (constant, scalar) is the parametricity of the measure __Measure.NONPARAMETRIC__.
+%%%% ¡default!
+Measure.NONPARAMETRIC
+
+%%% ¡prop!
+COMPATIBLE_GRAPHS (constant, classlist) is the list of compatible graphs.
+%%%% ¡default!
+{'MultiplexBU'} ;%TBE % % % add any missing tests
```

Comment:
Add all the other graphs that are compatible (all multilayer, multiplex, ordml and ordmx)